helm/nessie/values.yaml

```diff
@@ -37,6 +37,25 @@ authentication:
   # Set the OIDC client ID when authentication.enabled=true. Each application has a client ID that is used to identify the application
   oidcClientId: nessie
 
+# this is needed when selecting TRANSACTIONAL
+postgres:
+  secret:
+    name: postgres-creds
+    usernameKey: username
```

Comment:
nit: let's rename those to `username=postgres_username` and `password=postgres_password` as the `Key` suffix is rather confusing. This also requires some additional docs in the README, similar to how we have it for DYNAMO (see my example below). 


### Providing secrets for Postgres

* Make sure you have a Secret in the following form:
```
> cat $PWD/postgres-creds
postgres_username=YOUR_USERNAME
postgres_password=YOUR_PASSWORD
```

Reply:
With the `Key` suffix I meant to clarify that you need to specify the name of the key in the secret rather than the actual username or password but I can change it.

Reply:
yeah but when referring to secrets you're generally always referring to the Key within that secret, thus no need to use the `Key` suffix. With AWS secrets it's obviously different because the `Key` suffix is already part of the name "awsSecretAccessKey"